### ISSUE TITLE
`fig update` fails due to incorrect API hostname

### ISSUE THREAD

**Issue by bendrucker** [withfig/fig#556]
### Sanity checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue

### Description

Fig fails to load completion specs:

```
$ fig update

Pulling most up-to-date completion specs...
Run fig docs to learn how to contribute your own!

curl: (6) Could not resolve host: api.fig.io.com
tar: specs: Not found in archive
tar: Error exit delayed from previous errors.
```

Note `api.fig.io.com`, the addition of `.com` seems to be the issue here. Oddly `fig list`  works and seems to also use `curl` to load data.

### Environment

```bash
Version 1.0.51 (B294) [U.S.] 
UserShell: /bin/zsh
Bundle path: /Applications/Fig.app
Autocomplete: true
Settings.json: true
CLI installed: true
CLI tool path: /Users/ben/.fig/bin/fig
Accessibility: true
Number of specs: 1
SSH Integration: false
Tmux Integration: false
Keybindings path: <none>
iTerm Integration: true 
Hyper Integration: false
VSCode Integration: true
Docker Integration: true
Symlinked dotfiles: true
Only insert on tab: false
Installation Script: true
PseudoTerminal Path: /Users/ben/.rbenv/shims:/usr/local/opt/python/libexec/bin:/Users/ben/src/go/bin:/usr/local/Caskroom/google-cloud-sdk/latest/google-cloud-sdk/bin:/usr/local/bin:/usr/local/sbin:/Users/ben/.dotfiles/bin:/usr/local/opt/python/libexec/bin:/Users/ben/src/go/bin:/usr/local/Caskroom/google-cloud-sdk/latest/google-cloud-sdk/bin:/usr/local/bin:/usr/local/sbin:/Users/ben/.dotfiles/bin:/usr/local/opt/python/libexec/bin:/Users/ben/src/go/bin:/usr/local/Caskroom/google-cloud-sdk/latest/google-cloud-sdk/bin:/usr/local/bin:/usr/local/sbin:/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Applications/Visual Studio Code.app/Contents/Resources/app/bin:/Users/ben/.fig/bin:/Applications/Visual Studio Code.app/Contents/Resources/app/bin:/Applications/Visual Studio Code.app/Contents/Resources/app/bin
SecureKeyboardInput: false
SecureKeyboardProcess: <none>
Current active process: /usr/local/Cellar/zsh/5.8_1/bin/zsh (64512) - ttys005
Current working directory: /Users/ben/.dotfiles
Current window identifier: 8393/% (com.googlecode.iterm2)
PATH: /Users/ben/.rbenv/shims:/usr/local/opt/python/libexec/bin:/Users/ben/src/go/bin:/usr/local/Caskroom/google-cloud-sdk/latest/google-cloud-sdk/bin:/usr/local/bin:/usr/local/sbin:/Users/ben/.dotfiles/bin:/usr/local/opt/python/libexec/bin:/Users/ben/src/go/bin:/usr/local/Caskroom/google-cloud-sdk/latest/google-cloud-sdk/bin:/usr/local/bin:/usr/local/sbin:/Users/ben/.dotfiles/bin:/usr/local/opt/python/libexec/bin:/Users/ben/src/go/bin:/usr/local/Caskroom/google-cloud-sdk/latest/google-cloud-sdk/bin:/usr/local/bin:/usr/local/sbin:/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Applications/Visual Studio Code.app/Contents/Resources/app/bin:/Users/ben/.fig/bin:/Applications/Visual Studio Code.app/Contents/Resources/app/bin:/Applications/Visual Studio Code.app/Contents/Resources/app/bin
FIG_INTEGRATION_VERSION: 5
```

**Comment by bendrucker** [comment]
By the way, I initially ran this while trying to figure out why I wasn't seeing Fig completions for `git`.

**Comment by mschrage** [comment]
@bendrucker are the completions for `git` showing up now or are you still running into issues?

**Comment by bendrucker** [comment]
Yes, and I wish I had a solid of explanation of what changed/when. 

But I can at least say that the bundled app doesn't match what's here:

https://github.com/withfig/config/blob/main/tools/cli/update.sh

In `/Applications/Fig.app/Contents/Resources/config/tools/cli/update.sh` I have:

```
#MAGENTA=$(tput setaf 5)
#BOLD=$(tput bold)
#NORMAL=$(tput sgr0)
fig update:app --force
printf "\nPulling most up-to-date completion specs...\n"
printf "Run $(tput setaf 5)$(tput bold)fig docs$(tput sgr0) to learn how to contribute your own!\n\n"
# Download all the files in the specs folder of this repo

base_url='https://api.fig.io.com/specs';

# This is the current version of autocomplete
# It should be 1 ahead of the most recent branch we have created on github
# ie if we have a branch for v1 on withfig/autocomplete, make this v2
current_version=$(defaults read com.mschrage.fig "autocompleteVersion" 2>/dev/null) ;
build=$(defaults read com.mschrage.fig "build" 2>/dev/null) ;
app_version=$(fig --version) ;
mkdir -p ~/.fig/autocomplete; cd $_

curl -s "$base_url?version=$current_version&app=$app_version&build=$build" | tar -xz --strip-components=1 specs
```

**Comment by mschrage** [comment]
@bendrucker `fig update` doesn't actually have any effect on completion specs. This is all handled by the autocomplete-engine now.

**Comment by mschrage** [comment]
This is fixed as of v1.0.52!
